src/core/server/base.js

```diff
@@ -65,6 +66,9 @@ export default function(routes, createStore) {
 
       const store = createStore();
 
+      store.dispatch({type: 'SET_JWT', payload: {token: cookie.load('jwt_api_auth_token')}});
```

Comment:
If we support auth in other parts of the site we'll need it to I think it makes sense.

If you mean that I'm creating the action by hand then probably not. I should move it to `core/actions/index.js`.